docs/introduction.rst

```diff
@@ -1,73 +1,47 @@
-- Open source, cross platform (macOS, Windows, Linux), and Python based scientific data processing.
-- Low latency, high throughput data acquisition platform.
+User Interface
+==============
+The user interface is comprised of a central section called the workspace and the surrounding area for utility panels. 
 
-Data Items and Display Items
-----------------------------
-|AppName| operates on data arrays, which include 2D images and 1D spectra, which are generally called *Data Items*.
+The central workspace is for display panels which can either display a data item or can display thumbnail images of multiple data items side by side in one of the browser formats. For more information, see :ref:`display-panels`.
 
-Data items represent data arrays with anywhere from one to five dimensions. The data array associated with a data item can be 1D or 2D data and then organized into 1D or 2D collections or 1D sequences.
+The utility panels surrounding the central workspace allow you to make adjustments to a selected data item in a display panel. For more information on their specific functions, see :ref:`user-interface`.
 
-A spectrum image would be a 2D collection of 1D data. This would be typical data acquired from a STEM microscope spectrum imaging acquisition.
+Data and Display
+================
+Data is stored in a project as data items. Data items can be dragged into display panels to be displayed. Data items that are displayed in a display panel can be manipulated with graphics, computations, and more. These displayed data items are referred to as display items. Display items can be modified and viewed to see various aspects or values of the original data item. 
 
-Data Items are displayed using one or more *Display Items*.
+Read more in :ref:`data-items`.
 
-See :ref:`data-items`.
+Graphics and Masks
+==================
+Nion Swift provides plenty of useful graphics to narrow down your view of the data. Use graphics to highlight portions of the data, select areas for processing, or to focus on specific sections of data.
 
-Projects
---------
-Data items are organized into projects. Projects are stored on disk and take the form of an index file and an associated data directory.
+Masking allows you to block out the rest of an image and manipulate only the data you want. Choose from a variety of mask shapes to precisely define your area of interest.
 
-You can have any number of projects, but only a single one can be open at once. When you open a project, it is associated with a main document window.
+See :ref:`graphics` for more information.
 
-See :ref:`data-management`.
+Processing and Analysis
+=======================
+Processing data by adding line profiles to an image, running Python scripts, filtering data, etc. can give you deeper insight into your data. Use the :ref:`inspector panel` to adjust various settings to tailor processes to your specific needs. Customizing processes allows for easier and more specific analysis of data.
```

Comment:
"tailor processes to" -> "adjust processing for"

"Customizing processes..." -> "Customizing processing..."

Use the term "processing"